tephra-hbase-compat-1.1-base/src/main/java/org/apache/tephra/hbase/SecondaryIndexTable.java

```diff
@@ -177,11 +173,23 @@ public void close() throws IOException {
     } catch (IOException e) {
       try {
         secondaryIndexTable.close();
+        conn.close();
```

Comment:
`conn` may not be closed if `secondaryIndexTable.close()` throws an exception.

Adding another nested try-catch will make the code complex. Instead what do you think about putting each close in its own try-catch block, remember the first exception, and add the other exceptions as suppressed to the first exception?